test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -636,3 +636,80 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         $result.Error | Should BeNullOrEmpty
     }
 }
+
+Describe "Bug fixes" -Tags "Feature" {
+
+    Context "InFile parameter negative tests" {
+
+        $testCases = @(
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile ""'
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile ""}
+                ExpectedFullyQualifiedErrorId = 'WebCmdletInFileNotFilePathException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile'
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile}
+                ExpectedFullyQualifiedErrorId = 'MissingArgument,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile c:\pathThatDoesNotExist\foo.txt'
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile $HOME\foo.txt}
```

Comment:
> $HOME\foo.txt

It's very possible such as file might be created by other tests which don't clean up properly. This test would fail in that case.